0015-ilp-addresses/0015-ilp-addresses.md

```diff
@@ -54,105 +61,102 @@ The allocation scheme is the first part of an address, which indicates how the a
 
 - [Neighborhoods](#neighborhoods)
-- [Payment rail](#payment-rail)
-- [Ledger prefix](#ledger-prefix)
-- [Subledger and account group](#subledger-and-account-group)
-- [Account identifier](#account-identifier)
+- [Ledger prefixes](#ledger-prefixes)
+- [Account identifiers](#account-identifiers)
 - [Interactions](#interactions)
 
 Not all addresses contain all this information, and some addresses may use multiple segments to represent some of this information.
 
 ### Neighborhoods
 
-Neighborhoods have no specific meaning, but serve as a quick shortcut for routing to the right area.
+Neighborhoods have no specific meaning, but serve as a quick shortcut for routing to the right area. At this time, there is no official list of neighborhoods, but the following list of examples should illustrate what might constitute a neighborhood:
+
+- `crypto.` for ledgers related to decentralized crypto-currencies such as Bitcoin, Etherium, or XRP.
+- `sepa.` for ledgers in the [Single Euro Payments Area](https://en.wikipedia.org/wiki/Single_Euro_Payments_Area).
+- `dev.` for Interledger Protocol development and early adopters
 
-- For regional currencies, use a continent code from the following table:
-    | Segment | Continent     |
-    |:--------|:--------------|
-    | `af`    | Africa        |
-    | `as`    | Asia          |
-    | `eu`    | Europe        |
-    | `na`    | North America |
-    | `sa`    | South America |
-    | `oc`    | Oceania       |
-    | `an`    | Antarctica    |
-- For decentralized crypto-currency ledgers, use the `crypto` neighborhood.
-- For naturally-ocurring commodities or other globally-distributed currencies, use the `earth` neighborhood.
-- For currencies strongly associated with a particular international corporation, use the continent of that corporation's headquarters.
+Large neighborhoods can be further subdivided into nested sub-neighborhoods; for example, `dev.luxembourg.` for development ledgers hosted in Luxembourg.
 
-It may also be useful to have 1-3 levels of "sub-neighborhoods" to group together related connectors or ledgers. We have no specific recommendations at this time.
 
-### Payment Rail
+### Ledger Prefixes
 
-The payment rail refers to a specific way some accounts or ledgers are connected outside of ILP, such as ACH in the U.S., SEPA in Europe, or other online payment systems such as PayPal or Alipay. We recommend using an all-lowercase abbreviation for the payment rail if one is applicable to the address. Examples:
+A ledger prefix is the unique name of the ledger. When doing a local delivery, the [ilp-connector][] reference implementation uses this segment to figure out which ledger plugin to use. As a result, it's difficult (but possible) to make a connector that directly interacts with two ledgers that use the same ledger prefix.
```

Comment:
This "difficult (but possible)" comment is confusing. Ledgers should have different identifiers